app/helpers/permission-map.js

```diff
@@ -0,0 +1,18 @@
+import Ember from 'ember';
+
+/**
+* permissionMap helper.  Maps short form of the contributor's permissions
+* passed in params to the long form.
+*
+*/
+export function permissionMap(params/*, hash*/) {
+    var map = {
```

Comment:
There are already named constants in ember-osf that might be useful. We'd like to centralize this sort of thing where possible.
https://github.com/abought/ember-osf/blob/14afd9a7974573d04cab3efc379a3504218a5942/addon/const/permissions.js#L12-L12

The advantage of putting the constants elsewhere is that they can be used for other purposes as well, like rendering a select box/ dropdown for permissions.